### PR TITLE
Created a store using redux and redux saga

### PR DESCRIPTION
**Pull request by ArpitaDwivedi18**
In order to create redux store created different files under hooks and UI folder as follows (For API call functionality, redux saga is included) :
hooks
 -ReducerHub
   -action.tsx
   -reducer.tsx
   -store.tsx
   ...
UI
 -components
   -ForTestingStore
    -Header.tsx
    -Product.tsx
    ...

Documentation Link:
https://docs.google.com/document/d/1ARSbnkGnT_NxAG2M0mM58xqxznqgJU7Rx9U59qI78Wo/edit?usp=sharing

### CODE REVIEW

File: patches/axios+1.7.2.patch
Comment (Aashirvaad-P) [inline patches/axios+1.7.2.patch:1]: I haven't seen anything before being pushed. This is not needed to be pushed. Delete once and check.

File: src/Assets/image/logo.png
Comment (Aashirvaad-P) [inline src/Assets/image/logo.png:1]: Did you add it or was it added prior only?

**Comment by Aashirvaad-P** [comment]
Update your branch. Rest all seems good.

File: src/Assets/image/logo.png
Comment (ArpitaDwivedi18) [reply src/Assets/image/logo.png:1]: I think I added it earlier for the implementation of some other part, removed it now.

File: patches/axios+1.7.2.patch
Comment (ArpitaDwivedi18) [reply patches/axios+1.7.2.patch:1]: Deleted it, everything is working fine.

File: package.json [hunk 1]
```diff
@@ -7,20 +7,40 @@
...
+    "postinstall-postinstall": "^2.1.0",
+    "react": "18.2.0",
+    "react-native": "0.74.1",
+    "react-native-axios": "^0.17.1",
+    "react-native-safe-area-context": "^4.10.5",
+    "react-native-screens": "^3.32.0",
+    "react-redux": "^9.1.2",
+    "redux": "^5.0.1",
+    "redux-saga": "^1.3.0"
+=======
```
Comment (Aashirvaad-P) [inline package.json]: Fix this

File: package-lock.json
Comment (Aashirvaad-P) [inline package-lock.json:1]: This one as well, but it should be automatically fixed as you fix package.json.

**Review by Aashirvaad-P — APPROVED** [review]

**Pull request merged**